airflow/utils/helpers.py

```diff
@@ -317,3 +317,45 @@ def render_log_filename(ti, try_number, filename_template):
                                     task_id=ti.task_id,
                                     execution_date=ti.execution_date.isoformat(),
                                     try_number=try_number)
+
+
+def tail_file(filepath, lines):
```

Comment:
I would like to see a test of this function please, at least covering one happy/working case.

Reply:
Added. Please check